pkg/aws/client/client.go

```diff
@@ -118,6 +118,27 @@ func (c *Client) GetInternetGateway(ctx context.Context, vpcID string) (string,
 	return "", fmt.Errorf("no attached internet gateway found for vpc %s", vpcID)
 }
 
+// VerifyVPCAttributes checks whether the VPC attributes are correct.
+func (c *Client) VerifyVPCAttributes(ctx context.Context, vpcID string) error {
```

Comment:
I would vote to do not mess the awsclient pkg with requirements specific to the infra actuator. We can have a `DescribeVPCAttribute` func in the client but the real checks to happen in the actuator as the checks are specific to the actuator.

Reply:
Hm, not sure, I can do it this way if you insist, no problem, but then I'd question this client package at all, i.e., why do we need it if we already have the EC2 client etc (i.e., why do we need to wrap it again if we not unpack our specifics in it)?

Reply:
We already have Gardener-Extension-AWS-specific code in this client, see https://github.com/gardener/gardener-extension-provider-aws/blob/master/pkg/aws/client/client.go#L144 and following functions.